blueprints/ember-cli-qunit/index.js

```diff
@@ -7,7 +7,6 @@ module.exports = {
 
   afterInstall: function() {
     return this.addBowerPackagesToProject([
-      { name: 'ember-cli-test-loader',           target: '0.2.1'   },
```

Comment:
Should I update this to now add `ember-cli-test-loader` to the NPM dependencies?